schema/schemaspec/spec.go

```diff
@@ -9,10 +9,11 @@ import (
 type (
 	// Resource is a generic container for resources described in configurations.
 	Resource struct {
-		Name     string
-		Type     string
-		Attrs    []*Attr
-		Children []*Resource
+		Name      string
+		Qualifier string
```

Comment:
I'm thinking, maybe the qualifiers should be a type `[]string` instead? 
https://mariadb.com/kb/en/identifier-qualifiers/